docs/tutorials/ensuring-data-quality.mdx

```diff
@@ -47,41 +47,41 @@ Init for run commands:
 
         ``` yml workspace.sdf.yml
         # Uncomment here to begin the "Ensuring Data Quality" tutorial >>>>>>
-        defaults: 
+        defaults:
           preprocessor: jinja
         # <<<<<<
         ```
-        
-        For the sake of this tutorial, we will add tests on the `inapp_events` staging model located in 
+
+        For the sake of this tutorial, we will add tests on the `inapp_events` staging model located in
         `models/staging/inapp_events`.
-        
+
         To add tests to the model, we need to create a YML file to hold the model's metadata.
 
         Under `metadata/staging`, create a new file called `inapp_events.sdf.yml` containing the following definition:
 
         ``` yml metadata/staging/inapp_events.sdf.yml
         table:
-            name: app_installs
```

Comment:
The table name was probably incorrectly specified here.

If it's configured as `app_installs` here, where does it show up in my CLI workflow?

Reply:
Yep! Nice catch. This is correct